modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -417,6 +424,32 @@ private Product _randomProductWithSku() throws Exception {
 		};
 	}
 
+	private Product _randomVirtualProduct() throws Exception {
```

Comment:
what about _randomProductVirtual? it's quite strange that we have `_testPostProductVirtual()` and `_randomVirtualProduct()`